corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html

```diff
@@ -49,6 +49,26 @@ <h1>{% trans "Searching the Case Lists" %}</h1>
   </div>
 
   <div id="report-datagrid">
+
+    <div class="form form-horizontal well"
+         data-bind="foreach: reportFilters">
+      <div class="form-group">
```

Comment:
If you want the usual proportions, you can use `{% css_field_class %}` and `{% css_label_class %}`.